### ISSUE TITLE
Google Api key (not free? in this case)

### ISSUE THREAD

**Issue by Rachel9702** [ianseifs/Treepedia_Public#13]
Hello, I am a student who is learning hard even though I am not good at coding.
I have a question, so I am writing.
From what I heard, we have to pay to access more than a few Google Street views image from 2018. 
In the case of this code, does it cost money to get an API key issued and apply it? 
If anyone has used this code since 2019, please reply.
I need your help.... Thank you.

**Comment by sjeonglee** [comment]
Yes, you have to get an API key to access Google Street view images. Google provides some free credits for the first time if you join the cloud platform. If you use the credits, you may save some money.

Hope this helped!

**Comment by Rachel9702** [comment]
> Yes, you have to get an API key to access Google Street view images. Google provides some free credits for the first time if you join the cloud platform. If you use the credits, you may save some money.
> 
> Hope this helped!

Thank you So much!! 
Have a nice day~

**Comment by y26805** [comment]
Hi @Rachel9702 , I have a section in my fork's README that may help you. I explained how much the API may cost
https://github.com/y26805/Treepedia_Public#using-the-google-maps-api
